src/api-service/__app__/onefuzzlib/tasks/config.py

```diff
@@ -411,6 +411,13 @@ def build_task_config(
             else True
         )
 
+    if TaskFeature.preserve_existing_outputs in definition.features:
+        config.preserve_existing_outputs = (
+            task_config.task.preserve_existing_outputs
+            if task_config.task.preserve_existing_outputs is not None
+            else True
```

Comment:
Is this the right default? For example, on the CLI side, we [default to `False`.](https://github.com/microsoft/onefuzz/blob/main/src/cli/onefuzz/templates/libfuzzer.py#L399)

@chkeita, thoughts?

Reply:
It is the wrong default. I copied the code for `check_fuzzer_help` {which defaults to True} and forgot to change the default value.

Reply:
I've updated the pull request.